src/site/content/en/blog/sign-in-form-best-practices/index.md

```diff
@@ -562,7 +562,7 @@ implementation](https://twitter.com/sw12/status/1251191795377156099). You'll
 need to use JavaScript instead.
 
 <figure class="w-figure">
-  <img class="w-screenshot" src="./show-password-google.png" alt="Google sign-in form showing Show password icon." width="350">
+  ![Google sign-in form showing Show password icon.](./show-password-google.png)
```

Comment:
looks like this image was missed

Reply:
It looks like al lot of images from this article got missed. I'll get them in another round.